src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Sources/Result/FolderItemResult.cs

```diff
@@ -36,7 +37,7 @@ public Wox.Plugin.Result Create(IPublicAPI contextApi)
             {
                 Title = Title,
                 IcoPath = Path,
-                SubTitle = "Folder: " + Subtitle,
+                SubTitle = string.Format(CultureInfo.InvariantCulture, Properties.Resources.wox_plugin_folder_select_folder_result_subtitle, Subtitle),
```

Comment:
Shouldn't this be CultureInfo.CurrentCulture since we are localizing?   Even though it probably doesn't matter in this case.

Reply:
I copied it from other places in the program, but you have a valid point. I will change all the values to CurrentCulture when I'm home.

Reply:
Changed it to CurrentCulture for all instances